src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
Apparently only me. :)  I was thinking it would be useful to distinguish between something added later, such as MutationObserver which iirc is dom level4, but just calling them dom is probably sufficient, unless they are something that is changed across DOM versions (the only thing that comes to mind is deprecated mutation events).